Source/Private/Render/Windows/MainWindow.cpp

```diff
@@ -32,28 +32,38 @@ bool MainWindow::IsEnabled()
 
 void MainWindow::Draw()
 {
-    ImGui::Begin("Kyber", &m_isEnabled, ImGuiWindowFlags_AlwaysAutoResize);
-    ImGui::Checkbox("Show Menu [INSERT]", &m_isEnabled);
-    ImGui::Checkbox("Show Credits", &m_creditsWindow->m_isEnabled);
-    ImGui::Separator();
+    ImGui::Begin("KYBER", &m_isEnabled, ImGuiWindowFlags_AlwaysAutoResize);
+
     if (!g_program->m_server->m_running)
     {
-        if (ImGui::Button("Start a Server"))
+        if (ImGui::Button("SERVER SETUP"))
```

Comment:
If we're going to combine "Start a Server" and "Server Settings" then there's no point for this if statement, it's running the same code in both clauses.

Reply:
Not fixed, you need to remove the top-level server running check.